### PR TITLE
[JENKINS-67225] replace "default value" with "Set by Default"

### PR DESCRIPTION
**Pull request by jtnord**
fixes the UI where it appeared as though the checkbox was controlling
the option to set a default value rather than actually setting the
default value.

This does not update the localized versions - as I do not know the
correct terminology and rather than risk loosing the translation
altogether I have kept the original text

Not expecting any ATH issues as this just updates the text and not the control ([ref](https://github.com/jenkinsci/acceptance-test-harness/blob/a9027d40be247fbc51f4ea4af7e4bd0bd7f7e6e8/src/main/java/org/jenkinsci/test/acceptance/po/Parameter.java#L36-L39))

### Before

image

### After

image

See [JENKINS-67225](https://issues.jenkins-ci.org/browse/JENKINS-67225).

### Proposed changelog entries

* The wording for setting for the default value of a boolean job parameter was updated

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/hudson/model/BooleanParameterDefinition/config_it.properties
Comment (jtnord) [inline core/src/main/resources/hudson/model/BooleanParameterDefinition/config_it.properties:4]: 🤦

Comment (daniel-beck) [reply core/src/main/resources/hudson/model/BooleanParameterDefinition/config_it.properties]: Italian localization files have a messed up encoding, better to not attempt editing.

Comment (jtnord) [reply core/src/main/resources/hudson/model/BooleanParameterDefinition/config_it.properties]: should be fixed in the most recent update.

**Review by jglick — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
